### DISCUSSION TITLE
blog/retrospect/7-toss-interview

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#83] [General]
# blog/retrospect/7-toss-interview

과제 탈락에서 과제 칭찬 피드백을 받기까지 노력한 과정과 면접 복기를 통해 앞으로 개선할 점을 담은 내용을 기록하였습니다.

https://www.enjoydev.life/blog/retrospect/7-toss-interview

**Comment by choi-ik** [comment]
wow 오랜만에 수현님 블로그 놀러왔는데 이런 좋은 글이,, 이번 글을 통해 많은 것을 배워가네요 ㅎㅎ
최근 저도 과제 전형에서 미끄러진적이 있었는데, 단지 '요구사항 불충족' 이라고만 생각했었어요. 하지만 이렇게 단순하게 생각하면 안되겠다는 걸 수현님 글을 보고 느끼게 되네요 ㅎㅎ

수현님의 여러글들 보며 많이 배워갑니다 화이팅 !! 👍🏻

**Reply by pySoo** [reply]
헉 익님 오랜만이에요! 부족한 글이지만 좋게 봐주셔서 감사합니다.. ☺️
저도 아직 과제도 면접도 참 어렵게 느껴지지만... 매 경험마다 다음 도전에 큰 도움이 되는 것 같아요
익님의 이번 경험도 앞으로의 기회들을 잡기 위한 좋은 밑거름이 될 것이라고 생각합니다!

그리고 저도 데브코스분들 블로그 종종 놀러가는데 익님 글도 인상깊게 읽었어요!
특히 성능 최적화랑 개발 경험 향상 글 읽으면서 시도해보고 싶은 것들이 많이 생겼던 기억이 나네요 ㅎㅎ 저도 덕분에 많이 배웠습니다!! 👍

**Comment by devFancy** [comment]
당근 면접에 대한 글부터 여기까지 우연히 찾아온 백엔드 개발자에요. 🙂
수현님의 글을 보면서 '정말 열심히 살아가고 계시구나'를 느꼈고, 본인이 경험했던 것을 다시 한번 복기하면서 포스팅에 올리시는 모습을 보고 정말 대단하다고 느꼈어요.
제 경력은 아직 많지 않아서 경력직은 아니지만, 신입/인턴 공고에 지원하면서 느낀 부분들이 이 글을 통해 리마인드되면서 공감되고 재밌게 읽었어요.
수현님의 앞날을 응원합니다!! 👏🏻

**Reply by pySoo** [reply]
안녕하세요! 아직 부족한 글이지만 좋은 인상을 받으셨다니 기쁩니다 🥲
사실 저의 생각이 많이 담긴 글일수록 공개하기 전 두려움이나 주저함이 많았습니다.. ㅎㅎ
하지만 준용님의 댓글이 앞으로 글을 작성하는데 큰 동기부여가 될 것 같아요! 정말 감사합니다 🙇‍♀️

저도 준용님의 글 재밌게 읽었습니다!
글또의 다양한 활동들을 잘 활용하셔서 취업까지 이루어내신 모습이 정말 대단하고 멋지십니다 ㅎㅎ
성장하신 회고글을 읽으면서 덕분에 저도 글또 10기에 도전해 보아야겠다는 새로운 목표도 세울 수 있었어요
앞으로 준용님의 새로운 글들도 기대하겠습니다! 응원도 감사해요 👍

**Comment by yejinleee** [comment]
수현님 :) 구글링중에 좋은 글을 보았는데 수현님 블로그여서 구경하다가 이글을 보게되었어요 !!
이전과 비교해서 또 더욱 성장하고 계신것 같아서 멋집니다..!
또, 스스로 피드백하시는것과 마인드셋을 많이 배우게되는 글이었어요 체고,,
정말 고생하셨어요 응원합니다아 !!!!🧚‍♀️🧚‍♀️

**Reply by pySoo** [reply]
헉 예진님 너무 반가워요 😭 잘 지내고 계신가요?!
부족한 글들이지만 좋게 봐주셔서 감사합니다.. ㅎㅎ 🙇‍♀️
저도 예진님 야간등대 참여하시는 모습 보면서 자극도 많이 받고 꾸준함을 정말 본받고 싶다고 생각했어요..!
야간등대도 이젠 추억의 단어가 됐네요 🥲

다음에 여유되실 때 돌돌현분들이랑 같이 커피챗 나눠요~~
잊기 전에 제가 꼭 먼저 연락드릴게요! 예진님도 목표하신 것들 잘 이루어내시길 응원합니다!!! 🧚‍♀️🧚‍♀️

**Comment by taejinii** [comment]
우연히 들어오게되어 면접복기 글들을 보게되었는데 끈기가 대단하시네요~! 자극받고 갑니당
